src/main/java/org/bytedeco/javacv/FFmpegFrameGrabber.java

```diff
@@ -1325,6 +1329,27 @@ public Frame grabFrame(boolean doAudio, boolean doVideo, boolean doProcessing, b
                         frame.keyFrame = samples_frame.key_frame() != 0;
                     }
                 }
+            } else {
+                // Create and grow the streams array as necessary to avoid using HashMap 
```

Comment:
We can use `ArrayList` instead of an actual array to simplify this.

Reply:
Done for the array tracking the stream index to `Frame.otherStreams` (now `Frame.data`) mapping, array maintained for the Frame member for consistency with the image and samples arrays.